catalog/admin/module/catalog-admin-module-sources/pom.xml

```diff
@@ -109,22 +109,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
```

Comment:
Should probably leave a comment here. The reason why these all became zero is because I refactored the code that was being tested out of this module. There were only tests for that code and no tests for plugin.